krux_boto/util.py

```diff
@@ -39,3 +40,29 @@ def get_instance_region():
         get_logger('krux_boto').warn('get_instance_region failed to get the local instance region')
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
+
+
+# Region codes
+class RegionCode(Enum):
+    ASH = ('ASH', 'us-east-1')
+    PDX = ('PDX', 'us-west-2')
+    DUB = ('DUB', 'eu-west-1')
+    SIN = ('SIN', 'ap-southeast-1')
+
+    def __init__(self, code, region):
+        self.code = code
+        self.region = region
```

Comment:
I feel like one of these doesn't need to be required given you can derive one from the other?